pom.xml

```diff
@@ -427,6 +428,12 @@
                 <artifactId>config</artifactId>
                 <version>${config.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>org.influxdb</groupId>
+                <artifactId>influxdb-java</artifactId>
+                <version>${influxdb-java.version}</version>
+            </dependency>
```

Comment:
This dependency seems to be the MIT open source license, does it conflict with the Apache open source license? If there is no conflict, whether to add relevant instructions in the declaration file, please check. @CalvinKirs

Reply:
This is allowed, you can refer to this https://apache.org/legal/resolved.html

Reply:
> This is allowed, you can refer to this https://apache.org/legal/resolved.html

OK, Thanks